docs/src/pages/docs/Miscellaneous/issue_codes.mdx

```diff
@@ -113,3 +113,21 @@ The host might be down, and cannot be reached on the provided port.
 The host provided when adding a new database doesn't seem to be up.
 Additionally, it cannot be reached on the provided port. Please check that
 there are no firewall rules preventing access to the host.
+
+## Issue 1010
+
+```
+Superset found an error while running a command.
+```
+
+Something unexpected happened, and Superset encountered an error while
+running a command. Please reach out to your administrator.
+
+## Issue 1011
+
+```
+Superset found an unexpected error.
```

Comment:
nit: i prefer `encountered` here to `found`

```suggestion
Superset encountered an error while running a command.
```

Something unexpected happened, and Superset encountered an error while
running a command. Please reach out to your administrator.

## Issue 1011

```
Superset encountered an unexpected error.
```